plugin/ptrace/ptracewrappers.cpp

```diff
@@ -358,6 +368,13 @@ ptrace_single_step_thread(Inferior *inferiorInfo, int isRestart)
                             0);
     long inst = peekdata & 0xffff;
     if (inst == SIGRETURN_INST_16 && aarch64_regs.regs[0] == 0xf)
+#elif __riscv
```

Comment:
@aayushi363 , This code may be stale. But if we're going to add to it, then we definitely should add only correct code.  :-)
Please change to:
> `#elif defined(__riscv)`